.github/workflows/ballerinagentool.yml

```diff
@@ -0,0 +1,33 @@
+name: Ballerina PR validation
+on:
+  pull_request:
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Check out repository
+      uses: actions/checkout@v2
+    - name: Set up go
+      uses: actions/setup-go@v1
+      with:
+        go-version: '1.10.4'
+    - name: Execute the bash file
+      run: sudo sh ./ballerinaByExample/build-bbe.sh 
+
+    - name: Clone the ballerina-dev-website repo
+      run: git clone https://github.com/ballerina-platform/ballerina-dev-website.git
+    - name: Install Ruby
+      run: |
+        sudo apt update
+        sudo apt install ruby-full        
+    - name: Install Jekyll and bundler gems
+      run: sudo gem install jekyll
+    - name: Go to the cloned folder
+      run: cd ballerina-dev-website
```

Comment:
@sumangala028 
Here you need to update the workflow to include the generated BBE HTML files and commit and send a PR to ballerina-dev-website repository